extensions/proposals/EXT_texture_norm16/extension.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<proposal href="proposals/EXT_texture_norm16/">
+
+  <name>EXT_texture_norm16</name>
+
+  <contact> <a href="https://www.khronos.org/webgl/public-mailing-list/">WebGL
+  working group</a> (public_webgl 'at' khronos.org) </contact>
+
+  <contributors>
+  <contributor>Rijubrata Bhaumik (rijubrata.bhaumik 'at' intel.com)</contributor>
+    <contributor>Members of the WebGL working group</contributor>
+  </contributors>
+
+  <number>NN</number>
+
+  <depends>
+    <api version="2.0"/>
```

Comment:
1.0, unless there's a reason this can't be exposed for WebGL1.

Reply:
ext_texture_norm16 is written against the OpenGL ES 3.1, feature-wise corresponding to is WebGL 2. There is no technical reason to avoid WebGL1.